addons/l10n_sg_ubl_pint/models/account_tax.py

```diff
@@ -0,0 +1,28 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    # The classification is too precise to be able to do any kind of guessing, so we need it set.
+    # See https://docs.peppol.eu/poac/sg/2024-Q2/pint-sg/bis/#_goods_and_services_tax_gst
+    l10n_sg_ubl_pint_tax_category = fields.Selection(
+        selection=[
+            ('SR', 'Local supply of goods and services'),
```

Comment:
Same small suggestion feel free to do it, but here the key can stay in single quotes but the value could be in double :smile: 
Edit: specially since you do it for ubl_cii_format let's stay consistent